### PR TITLE
addpart: add page

### PR DESCRIPTION
**Pull request by jenesh**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

This PR is in support of #2214.

### CODE REVIEW

File: pages/linux/addpart.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# addpart
+
+> Tells the Linux kernel about the existence of the specified partition. The command is a simple wrapper around the "add partition" ioctl, [Docs](https://linux.die.net/man/8/addpart).
```
Comment (navarroaxel) [inline pages/linux/addpart.md:3]:
```suggestion
> Tells the Linux kernel about the existence of the specified partition.
> The command is a simple wrapper around the `add partition` ioctl.
> More information: <https://linux.die.net/man/8/addpart>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @jenesh, I left some comments.

File: pages/linux/addpart.md (hunk 1 above)
Comment (jenesh) [reply pages/linux/addpart.md]: Thank you for the suggested changes, it looks much cleaner now.

**Review by navarroaxel — APPROVED** [review]
LGTM

**Review by sbrl — APPROVED** [review]
Thanks, @jenesh!

**Pull request merged**